backend/diagnostics.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 )
 
+// CheckHealthHandler enables users to send health check
+// requests to a backend plugin
```

Comment:
Need to add a new `CheckHealthRequest` struct and change this interface to take struct as argument in `CheckHealth` method.

Reply:
ofc... I keep missing the required SDK changes :)

Should be fixed now